declarative-schedule-doc.md

```diff
@@ -339,6 +340,90 @@ scenarios:
 And on top of that you can use aliases to avoid duplication. For more details, check output
 [official documentation of openQA](http://open.qa/docs/)
 
+### 5. Use YAML_SCHEDULE_DEFAULT and YAML_SCHEDULE_FLOWS to YAML schedule
+
+
+We have 3 variables to help YAML scheduler to avoid conditional schedule and repetition. They are YAML_SCHEDULE_DEFAULT, YAML_SCHEDULE_FLOWS, YAML_SCHEDULE.
+
+Please be noted that conditional schedule does not work with the new scheduler that uses DEFAULT and FLOWS.
+
+YAML_SCHEDULE_DEFAULT should point to a "default" minimal schedule that has sections.  You can find them in schedule/yast/sle/flows/default.yaml. Sections can have "normal yaml schedule lines" or are just "empty sockets" (indicated by "[]").
+
+```yaml
+# Default ordered sequence of steps to be optionally overwritten for this product
+bootloader:
+  - installation/bootloader_start
+setup_libyui:
+  - installation/setup_libyui
+access_beta: []
+product_selection:
+  - installation/product_selection/install_SLES
+license_agreement:
+  - installation/licensing/accept_license
+
+```
+
+YAML_SCHEDULE_FLOW is a comma separated list of "flow" modules. like schedule/yast/sle/flows
```

Comment:
s/comma separated/comma-separated/

Reply:
please don't use any path like /schedule/yast/sle/flows, all off that might change, just point to the file name.